Animations/Fade/Right.js

```diff
@@ -26,5 +28,5 @@ var FadeRight = {
     'transform': 'translateX(20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeRight', FadeRight);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeRight', FadeRight);
```

Comment:
This does not need to have `'FadeRight'` passed into it since we have defined a `className` property in `FadeRight`.